explainer/README.md

```diff
@@ -85,15 +85,10 @@ Since fenced frames are embedded frames, they also behave like iframes in many w
 
 
 ```
-<fencedframe src="demo_fenced_frame.html"></fencedframe>
+var fencedframe = document.createElement('fencedframe');
+fencedframe.config = FencedFrameConfig({'src': 'demo_fenced_frame.html'});
```

Comment:
Right now the [ctor only takes a URL](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/html/fenced_frame/fenced_frame_config.idl;l=16;drc=d99163d64d43b85a9efcf0110a0cfa81886b4616), so maybe we should reflect that here?

Or if we really want to be able to pass in a dictionary of things, we need to make that implementation change now, before we roll any of these APIs out to the wild. If that's our intention, then we can leave this here and make an APIFreeze2-blocking task to update the IDL any and tests ASAP.

Reply:
Changed the constructor to pass a URL only.